modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/run-gitops-control-plane-workload-on-infra-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
+
+.Prerequisites
+* {gitops-title} is installed in your cluster.
```

Comment:
@reginapizza Are there any other prerequisites?

Reply:
we should add that you have to have the cluster-admin role here too actually